libs/community/langchain_community/indexes/_memory_recordmanager.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Sequence
+
+from langchain_community.indexes.base import RecordManager
+
+
+class MemoryRecordManager(RecordManager):
```

Comment:
We have it in unit tests right now

https://github.com/langchain-ai/langchain/blob/main/libs/core/tests/unit_tests/indexing/in_memory.py#L13-L13

Do you want to just migrate it out of unit tests and into langchain_core?

I'm in favor of having InMemory implementations directly available in either langchain_core or langchain

Reply:
done